pennylane/templates/layers.py

```diff
@@ -67,19 +69,15 @@ def StronglyEntanglingLayer(weights, wires, r=1, imprimitive=CNOT):
         r (int): range of the imprimitive gates of this layer, defaults to 1
         imprimitive (pennylane.ops.Operation): two-qubit gate to use, defaults to :class:`~pennylane.ops.CNOT`
 
-    Raises:
-        ValueError: if less than 2 wires were specified
     """
-    if len(wires) < 2:
-        raise ValueError("StronglyEntanglingLayer requires at least two wires or subsystems to apply "
-                         "the imprimitive gates.")
 
     for i, wire in enumerate(wires):
         Rot(weights[i, 0], weights[i, 1], weights[i, 2], wires=wire)
 
-    num_wires = len(wires)
-    for i in range(num_wires):
-        imprimitive(wires=[wires[i], wires[(i + r) % num_wires]])
+    n_wires = len(wires)
+    if n_wires > 1:
+        for i in range(n_wires):
+            imprimitive(wires=[wires[i], wires[(i + r) % n_wires]])
 
```

Comment:
This one is smuggled in from a later PR, but it makes the tests run with 1 wire as well...